dependencies.yaml

```diff
@@ -155,9 +155,6 @@ dependencies:
         packages:
           - ipython
           - notebook>=0.5.0
-      - output_types: [conda]
-        packages:
-          - wget
```

Comment:
`wget` already comes pre-installed in our CI images, you don't need the conda version of it.